recipes/blast/build.sh

```diff
@@ -15,13 +16,24 @@ if test x"`uname`" = x"Linux"; then
 fi
 
 if [ `uname` == Darwin ]; then
-    export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib -lz -lbz2"
+    #export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib -lz -lbz2"
+    export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib -lz -lbz2 -flat_namespace -headerpad_max_install_names"
+    export CXXFLAGS="$CXXFLAGS -fpascal-strings -fpermissive -Wno-deprecated-register -fno-common"
+    export CFLAGS="$CFLAGS -fpascal-strings -Wno-deprecated-register -fno-common"
```

Comment:
Can you explain (ideally with comments in the build script 😉) why those additional flags are added?

Reply:
Will be difficult, it was a while ago. I have tried hundred of combination to minimise warnings and error when compiling os OSX.
Those changes are maybe not needed anymore with the new SDK

Reply:
Okay, then I'll try a build without them.

Reply:
Yes it works now. Thx